admin/sql/create_tables.sql

```diff
@@ -8,4 +8,15 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 
+CREATE TABLE listens (
+  id              SERIAL,
+  uid             VARCHAR NOT NULL,
+  date            TIMESTAMP,
+  timestamp       INT NOT NULL,
```

Comment:
Why have two? you don't need a date field if you have a timestamp -- the timestamp includes the date. Please remove the date field and change TIMESTAMP to TIMESTAMP WITH TIME ZONE.